samples/kotlin-android-app/src/main/java/com/amplitude/android/sample/MainActivity.kt

```diff
@@ -0,0 +1,41 @@
+package com.amplitude.android.sample
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.amplitude.core.events.EventOptions
+import com.amplitude.core.events.Identify
+import com.amplitude.core.events.Plan
+import com.amplitude.core.events.Revenue
+import org.json.JSONObject
+
+class MainActivity : AppCompatActivity() {
+    private val amplitude = MainApplication.amplitude
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        // set user properties
+        val identify = Identify()
+        identify.set("user-platform", "android")
+            .set("custom-properties", "sample")
+        val options = EventOptions()
+        options.plan = Plan(branch = "test")
+        amplitude.identify(identify, options)
+
+        // set groups fro this user
+        amplitude.setGroup("test-group-type", "android-kotlin-sample")
+        amplitude.setGroup("orgId", "15")
+        amplitude.setGroup("sport", arrayOf("tennis", "soccer")) // list values
+
```

Comment:
groupIdentify?

Reply:
Good catch! Let me add example for groupIdentify